cmake/section_relocation.cmake

```diff
@@ -0,0 +1,11 @@
+function(zephyr_kernel_code_relocate file location)
+  get_property(CODE_DATA_RELOCATION_STRINGS_LOCAL GLOBAL PROPERTY
+   CODE_DATA_RELOCATION_STRINGS)
+  if(NOT ${CODE_DATA_RELOCATION_STRINGS_LOCAL} STREQUAL '')
+    set_property(GLOBAL APPEND_STRING PROPERTY CODE_DATA_RELOCATION_STRINGS ",")
+    set_property(GLOBAL APPEND_STRING PROPERTY CODE_DATA_RELOCATION_STRINGS ${location}:${file})
```

Comment:
Can you do this in one line?

set_property(GLOBAL APPEND_STRING PROPERTY CODE_DATA_RELOCATION_STRINGS ,${location}:${file})

Reply:
done, pls check